docs/legend.php

```diff
@@ -30,6 +32,10 @@
                     header("Location: facultyMap.php");
                 }elseif ($person_selected == 'visitor') {
                     header("Location: visitorMap.php");
+                }elseif ($person_selected == 'handicap') {
+                    header("Location: handicapMap.php");
+                }elseif ($person_selected == 'motorcycle') {
+                    header("Location: motorcycleMap.php");
                 }               
         }
```

Comment:
in legend.php, we need a way to go back to the main map. we can add this using
`<form method="post" action="map.php">
            <input type="submit" value="Back to main map"/>
        </form>`

This will allow the user to go back without having to restart the web app or use the back arrows